addons/website/data/image_library.xml

```diff
@@ -331,6 +331,12 @@
     <field name="type">url</field>
     <field name="url">/website/static/src/img/snippets_demo/s_team_member_4.jpg</field>
 </record>
+<record id="website.s_showcase_default_image" model="ir.attachment">
+    <field name="public" eval="True"/>
+    <field name="name">s_showcase_default_image.jpg</field>
+    <field name="type">url</field>
+    <field name="url">/website/static/src/img/snippets_demo/s_showcase.jpg</field>
```

Comment:
Good catch ! I've added a fallback 👍

Reply:
Getting this now:
![image](https://github.com/user-attachments/assets/89b82ed3-3643-4f6a-a3a3-26536509ceb1)

Should the icons contrast be improved (it seems it uses the preset's text color - but that's not on top of the preset's background color) ?
Should the image fill the column width ? (I think not - this is most likely ok because that's also the case with the default content)

Reply:
> it seems it uses the preset's text color

It seems this comes from a default background color defined here : https://github.com/odoo/odoo/blob/master/addons/website/static/src/scss/website.scss#L565

I've added classes to fix that ✅  : https://github.com/odoo/odoo/compare/2e81ca941206d662ffb410b987b31c1fa3b46e48..6644668c4f22dfbae90a91432dc666a809ef201b

> Should the image fill the column width

I don't think so, otherwise if the image is too small, it will be really blurred:
![Capture d’écran 2024-07-26 à 15 00 32](https://github.com/user-attachments/assets/59b198a2-37a4-4fae-868c-eac19c4e64ee)